README.md

```diff
@@ -63,18 +26,14 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235665753-c5edd2ff-32b8-4f05-902e-38dbc90bf942.png)

- [ ] It would be nice if you fix the links shown in the image above to make your readme descriptive and professional 😊